x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -454,6 +479,52 @@ export const InnerVisualizationWrapper = ({
     );
   }
 
+  if (localState.missingRefsErrors?.length) {
+    // Check for access to both Management app && specific indexPattern section
+    const { management: isManagementEnabled } = application.capabilities.navLinks;
+    const isIndexPatternManagementEnabled =
+      application.capabilities.management.kibana.indexPatterns;
+    return (
+      <EuiFlexGroup data-test-subj="configuration-failure">
+        <EuiFlexItem>
+          <EuiEmptyPrompt
+            actions={
+              isManagementEnabled && isIndexPatternManagementEnabled ? (
+                <EuiButtonEmpty
+                  onClick={() =>
+                    application.navigateToApp('management', {
+                      path: `/kibana/indexPatterns/create`,
+                    })
+                  }
```

Comment:
Should this be an `href`?

Reply:
That's a good point, it would be cool if this would be a link. @dej611 
You can use `getUrlForApp` instead of navigate to get the URL - then you need to wrap the component into a `<RedirectAppLinks>` - this will intercept the click and prevent the page load if it's not opened in a new tab.